packages/block-library/src/loginout/index.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * WordPress dependencies
+ */
+import { __, _x } from '@wordpress/i18n';
+import { postList as icon } from '@wordpress/icons';
```

Comment:
A new icon will be needed here.

Reply:
cc @jasmussen for the icon. Any ideas/suggestions?

Reply:
How about this?

<img width="379" alt="Screenshot 2021-03-17 at 11 54 01" src="https://user-images.githubusercontent.com/1204802/111456868-cb50cd80-8717-11eb-9365-8ef4bce8a1a6.png">

```
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M11 14.5l1.1 1.1 3-3 .5-.5-.6-.6-3-3-1 1 1.7 1.7H5v1.5h7.7L11 14.5zM16.8 5h-7c-1.1 0-2 .9-2 2v1.5h1.5V7c0-.3.2-.5.5-.5h7c.3 0 .5.2.5.5v10c0 .3-.2.5-.5.5h-7c-.3 0-.5-.2-.5-.5v-1.5H7.8V17c0 1.1.9 2 2 2h7c1.1 0 2-.9 2-2V7c0-1.1-.9-2-2-2z" fill-rule="evenodd" clip-rule="evenodd" /></svg>
```

Reply:
Perfect! Added in https://github.com/WordPress/gutenberg/pull/29766/commits/66327a3b9ec45fd43635140e194cb5ecf5f95242 👍 
Thank you Joen :)